Set-1/Solved Programs/KattapaReadyForBattle/KattapaReadyForBattle.java

```diff
@@ -0,0 +1,41 @@
+package com.problems;
+
+import java.util.logging.Logger;
+
+public class KattapaReadyForBattle {
+
+    public static String isReadyForBattle(int n,int[] soldierWeapons){
+    	
+    	/** The logger. */
+	   Logger logger = Logger.getLogger(KattapaReadyForBattle.class.getName());
```

Comment:
is it good practise to define logger at method level?

Reply:
No  its not good . need to declare outside the method